news-bundle/src/EventListener/SitemapListener.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\NewsBundle\EventListener;
+
+use Contao\Controller;
+use Contao\CoreBundle\Event\ContaoCoreEvents;
+use Contao\CoreBundle\Event\SitemapEvent;
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\News;
+use Contao\NewsArchiveModel;
+use Contao\NewsModel;
+use Contao\PageModel;
+use Terminal42\ServiceAnnotationBundle\Annotation\ServiceTag;
+
+/**
+ * @ServiceTag("kernel.event_listener", event=ContaoCoreEvents::SITEMAP)
```

Comment:
I think we're not using annotations but define our service tags inside the `.yml` file.
(also, we _should_ use PHP8 attributes for Contao 5 😆)

Reply:
What do you mean? We are using annotations for service tagging in countless services already. But yes, I can change it to PHP 8 attributes instead.